### PR TITLE
Shreeji818 patch 8

### PR DESCRIPTION
**Pull request by shreeji818**
Sync up Db2 workflows with workflows in /IBM/Db2ZTools/DevOps repository

### CODE REVIEW

**Review by Conner-IBM — CHANGES_REQUESTED** [review]
Just to clarify, this is how DB2 is currently shipping these workflows? Having a binary pax instinctively feels off when hosting on github, as it makes it difficult to view or contribute to the source files. However if this is the format that has been provided by the DB2 middleware team, I think I can understand their intended usecase.

However, the currently included .DS_store files need to be removed from this commit.

**Comment by shreeji818** [comment]
Added .gitignore to ignore .DS_Store files in commit.

**Comment by shreeji818** [comment]
Removed .DS_Store files from commit. Please review.

File: provisioning-workflows/Db2/README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Db2
-Place Holder
+Various workflows that provide Db2 provisioning services. To provision a new Db2 subsystem (DataSharing or non Data Sharing) refer to Db2SystemServices. To provision Db2 Schema and sample database refer to Db2SchemaServices.
```
Comment (Conner-IBM) [inline provisioning-workflows/Db2/README.md:2]: I think this file should be updated to explain the use of pax files in this repo.

**Review by Conner-IBM — APPROVED** [review]
DS_Store files where removed. 
We have discussed the approach about simply shipping pax files instead of readable files, and currently this is the approach DB2 is taking.
I think that the readme for this upper level directory could be improved to explain why these choices where made to help avoid confusion from anyone browsing this repository, aside from that cosmetic update- Looks good. So marking approved. Readme can be updated now or via git if preferred.

**Pull request merged**